rootfs/standard/usr/share/mynode_apps/datum/datum.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "DATUM",
+    "short_name": "datum",
+    "author": {
+        "name": "OCEAN",
+        "link": "https://ocean.xyz"
+    },
+    "website": {
+        "name": "GitHub",
+        "link": "https://github.com/OCEAN-xyz/datum_gateway"
+    },
+    "category": "bitcoin_app",
+    "short_description": "Self-sovereign Bitcoin mining",
+    "description": [
+        "DATUM (Decentralized Alternative Templates for Universal Mining) is a brand new bitcoin mining protocol that allows you to create your own block templates using your own Bitcoin node."
+    ],
+    "latest_version": "v0.2.2",
+    "supported_archs": ["aarch64","x86_64"],
+    "download_skip": true,
+    "download_type": "source",
+    "download_source_url": "https://github.com/OCEAN-xyz/datum_gateway/archive/refs/tags/v0.2.2beta.tar.gz",
+    "download_binary_url": {
+        "aarch64": "https://github.com/Retropex/datum_gateway/releases/download/v0.2.2beta/datum-0.2.2-linux-aarch64.tar.gz",
```

Comment:
You can leave the binaries empty if you are building rom source. The source tarball will automatically download and extract.